about.html

```diff
@@ -12,53 +12,64 @@
 </head>
 
 <body>
+  <div class="container">
+  <div class="navigation-bar">
+    <div class="lambda-logo">
+      <h1>ambda</h1>
+    </div>
+    <div class="nav-bar">
+        <a href="#">Home</a>
+         <a href="about.html">About</a>
+         <a href="#">Products</a>
+         <a href="#">Blog</a>
+         <a href="#">Contact</a>
+    </div>
+   
+   </div>
+<header>
+  <img src="img/jumbo.jpg" alt="jumbo" /> 
+  <h1>About</h1>
+          <p>
+          Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
+          </p>
+</header>
     <div class="container about-page">
-     
-    About
-
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
-
-
-    <img src="img/about-plan.png" alt="strategy">
-    
-    Strategy
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
-
-
-    <img src="img/about-working.png" alt="strategy">
-    
-    How We Work
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
-
-
-    <img src="img/about-office.png" alt="strategy">
-    
-    Places We Work
-
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
```

Comment:
The about page content is another great place to use semantic HTML, it can be laid out with sections.